packages/manager/modules/user-account/src/images/user-account/security/code.svg

```diff
@@ -0,0 +1 @@
-1.875-6.201-4.767-.114-.497-.14-1.021-.141-1.533-.008-22.089-.014-44.178-.003-66.267.002-3.84 2.664-6.509 6.457-6.513 11.474-.012 22.948-.004 34.421-.003.253 0 .506.025.869.045 0 3.511-.004 6.912.004 10.313 0 .426-.012.87.097 1.275.261.972 1.11 1.519 2.317 1.523 3.262.01 6.524.029 9.785-.014.791-.011.998.211.998 1.001-.022 19.285-.016 38.57-.016 57.855 0 .866.157 1.774-.048 2.59zm-8.441-71.517c1.014 1.044 1.887 1.967 2.787 2.862.888.884 1.804 1.742 2.709 2.611l-.141.21h-5.355v-5.683zm12.346 6.513c-4.297-4.318-8.607-8.621-12.919-12.925-.374-.374-.786-.712-1.179-1.066h-39.832c-1.256.441-2.61.709-3.749 1.355-3.702 2.098-5.313 5.448-5.317 9.624-.01 10.956-.006 21.913-.006 32.87 0 11.042.001 22.084.003 33.127.002 5.379 2.704 9.094 7.834 10.785l.205.142h47.901c.339-.131.675-.269 1.018-.392 4.276-1.531 7.012-5.471 7.019-10.183.013-9.666.007-19.332.007-28.998 0-10.638-.012-21.276.012-31.915.002-.989-.345-1.769-.997-2.424z" fill="#898989"/><path d="M59.377 33.029c-.013-4.122-1.521-7.64-4.402-10.558-3.198-3.237-7.19-4.137-11.575-3.831-6.623.464-11.735 5.128-12.227 11.729-.359 4.809-.246 9.668-.068 14.495.124 3.341 2.854 5.608 6.207 5.628 2.117.013 4.234.003 6.351.003h3.842v.913c.001 2.919-.007 5.837.014 8.756.003.383.065.815.25 1.138.393.692.865 1.342 1.343 1.98.322.427.342.781.016 1.222-.407.551-.755 1.146-1.111 1.732-.524.86-.526 1.731.006 2.59.391.632.819 1.243 1.178 1.892.114.206.18.581.07.746-.7 1.039-1.352 2.139-2.213 3.033-1.76 1.825-3.619 1.654-5.144-.38-.596-.795-1.09-1.669-1.64-2.52.531-.733 1.035-1.424 1.532-2.12.746-1.046.752-1.962.016-3-.508-.718-1.027-1.429-1.541-2.143.238-.33.406-.64.645-.881 1.454-1.465 1.734-3.261 1.563-5.219-.117-1.348-1.03-2.274-2.324-2.244-1.248.03-2.166.895-2.076 2.229.099 1.463-.302 2.679-1.201 3.791-.286.353-.509.76-.743 1.152-.497.831-.516 1.677-.026 2.517.473.811 1.336 1.619 1.33 2.422-.007.808-.994 1.564-1.323 2.429-.242.638-.406 1.58-.107 2.101 1.244 2.171 2.438 4.413 4.478 6.006 1.343 1.048 2.819 1.725 4.557 1.738 2.037.016 3.695-.86 5.167-2.177 1.736-1.553 2.845-3.577 3.966-5.559.297-.527.071-1.424-.088-2.111-.11-.473-.466-.925-.805-1.302-.71-.793-.704-1.509.003-2.299 1.477-1.648 1.441-3.318-.006-4.977l-.19-.285c-.824-.965-1.04-2.069-.996-3.331.083-2.424.023-4.853.023-7.346.304-.023.475-.053.644-.045 3.756.178 6.788-2.75 6.635-6.757-.14-3.714-.017-7.438-.03-11.157" fill="#898989"/><path d="M54.335 44.175c-.004 1.24-.673 1.893-1.929 1.902-2.545.018-5.091.006-7.636.006-2.489.001-4.978.008-7.466-.002-1.484-.005-2.076-.602-2.079-2.089-.006-4.032-.019-8.066.003-12.099.026-5.237 3.708-8.887 8.956-8.855 1.277.008 2.595.021 3.821.328 3.659.917 6.246 4.176 6.31 7.938.075 4.289.033 8.581.02 12.871" fill="#cbd0d5"/><path d="M44.71 26.974c-3.017-.018-5.432 2.293-5.523 5.283-.043 1.409.694 2.305 1.987 2.413 1.325.112 2.229-.68 2.372-2.111.069-.689.343-1.155 1.095-1.161.79-.006 1.076.456 1.133 1.2.105 1.354 1.061 2.178 2.331 2.074 1.224-.1 2.098-1.126 2.055-2.412-.098-2.947-2.492-5.269-5.45-5.286" fill="#898989"/></g></svg>
```

Comment:
maybe in a next step we could check if we can switch all those to `ui-kit` icons